index.js

```diff
@@ -916,7 +1115,18 @@ function importConverterGeneral(path, context) /*: K.Import */ {
       };
     }
 
-    let file = loadFileSync(filePath);
+    // If the moduleSpecific isn't a local/relative import don't load/resolve it.
+    // This is a workaround for export * not recursively being resolved.
+    if (moduleSpecifier.indexOf('.') !== 0) {
+      return {
+        kind: 'import',
+        importKind,
+        name,
+        moduleSpecifier
+      };
+    }
+
```

Comment:
Sorry, just noticed the consequence of this. We have use-cases such as `@atlaskit/avatar-group` where it imports types from `@atlaskit/avatar`, and we want to resolve them.

Ideally we do want to resolve types from other packages (when they can be found). We're going to need to rethink how to do this to solve for your recursivity problems.